snapfaas/bins/gateway/main.rs

```diff
@@ -0,0 +1,77 @@
+use clap::{App, Arg, crate_authors, crate_version};
+use std::sync::{Arc, Mutex};
+use std::net::SocketAddr;
+use std::thread;
+use snapfaas::sched::{
+    schedule_sync,
+    rpc::Scheduler,
+    gateway::{
+        Gateway,
+        HTTPGateway,
+        SchedGateway,
+    },
+    resource_manager::ResourceManager,
+};
+
+fn main() {
+    env_logger::init();
+
+    let matches = App::new("Faasten Gateway")
+        .version(crate_version!())
+        .author(crate_authors!())
+        .about("Launch Faasten gateway")
+        .arg(
+            Arg::with_name("http listen address")
```

Comment:
Is this listen address kept for the frontend `webfront`?

Reply:
I think so? This listens for the external invokes.

Reply:
Cool. Could you add a comment to this, saying something like `TODO upgrade webfront to use labeled_invoke and remove http listen address`?

Reply:
Fixed